pkg/crc/preflight/preflight.go

```diff
@@ -18,6 +18,10 @@ const (
 	StartUpOnly
 )
 
+var (
+	preset string
+)
+
```

Comment:
We've avoided global variables in the `preflight` package so far, would be better to keep it this way.